src/ol_infrastructure/lib/ol_types.py

```diff
@@ -25,6 +25,21 @@ class BusinessUnit(str, Enum):  # noqa: WPS600
     starteam = 'starteam'
     xpro = 'mitxpro'
 
+@unique
+class Apps(str, Enum):  # noqa: WPS600
+    """Canonical source of truth for defining apps."""
+
+    bootcamps = 'bootcamps'
+    micromasters = 'micromasters'
+    ocw_build = 'ocw-build'
+    redash = 'redash'
+    dagster = 'dagster'
+    mitxpro_edx = 'xpro-edx'
+    mitx_edx = 'mitx-edx'
+    odl_video_service = 'ovs'
```

Comment:
Should we have the value of this be `odl-video`?

Reply:
It can be, but seems to be internally identified now as `ovs`